packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -128,15 +128,7 @@ export class PendingStateManager implements IDisposable {
 			return {
 				pendingStates: [...this.savedOps, ...this.pendingMessages.toArray()].map(
 					(message) => {
-						let content = message.content;
-						const parsedContent = JSON.parse(content);
-						// IdAllocations need their localOpMetadata stashed in the contents
-						// of the op to correctly resume the session when processing stashed ops
-						if (parsedContent.type === ContainerMessageType.IdAllocation) {
-							parsedContent.contents.stashedState = message.localOpMetadata;
-							content = JSON.stringify(parsedContent);
-						}
-
+						const content = message.content;
```

Comment:
You can get rid of this variable and just have the line below be `return { ...message, localOpMetadata: undefined };` since we are no longer changing the `content` variable.

Reply:
Nice, updated in latest commit